### DISCUSSION TITLE
CLI flag for sentence level (srt) output

### DISCUSSION THREAD

**Discussion by dgoryeo** [linto-ai/whisper-timestamped#73] [Q&A] [answered]
Hi, 

Thanks again for your great work. A quick rudimentary question:  what is the CLI flag to be able to get sentence level srt output? 

Thanks

**Comment by Jeronymous** [comment, marked as answer]
There is no particular flag for this.
If "srt" is an output format (by default with `--output_format=all`, or with `--output_format=srt`, or with `--output_format=...,srt,...`), then two srt files will be generated for each input audio:
* one `audio.ext.words.srt` with the timestamps of the words
* one `audio.ext.srt` with the timestamps of the segments identified by Whisper (that might consists on a sentence, a set of sentences, or a part of sentences, depending on the length of the sentence).

If you need to have exactly one sentence per line, you might need to do a specific treatment using `audio.ext.words.srt`.
I guess you would use punctuation marks to identify ends of sentences (although it might be incorrect sometimes, e.g. with abbreviations "Mr. White is ...").

**Reply by dgoryeo** [reply]
Thanks for the good clarification @Jeronymous. Very helpful as always.
